source/_integrations/upnp.markdown

```diff
@@ -12,13 +13,15 @@ ha_codeowners:
 ha_domain: upnp
 ha_ssdp: true
 ha_platforms:
+  - binary_sensor
   - sensor
 ---
 
-The `upnp` integration enables you to collect network statistics from your router such as bytes in/out and packets in/out. This information is provided by the [UPnP](https://en.wikipedia.org/wiki/Universal_Plug_and_Play)/[Internet Gateway Device (IGD) Protocol](https://en.wikipedia.org/wiki/Internet_Gateway_Device_Protocol) if enabled on your router.
+The `upnp` integration enables you to collect network statistics from your router such as bytes in/out and packets in/out and information such as WAN connectivity, WAN IP, and uptime. This information is provided by the [UPnP](https://en.wikipedia.org/wiki/Universal_Plug_and_Play)/[Internet Gateway Device (IGD) Protocol](https://en.wikipedia.org/wiki/Internet_Gateway_Device_Protocol) if enabled on your router.
 
 There is currently support for the following device types within Home Assistant:
 
+- **Binary Sensor** - Router connected to WAN, attributes include actual status text, WAP IP, and uptime of router.
```

Comment:
How can a binary sensor include text?

Reply:
Was included as attributes. Removed now and will be added through other PR as new sensors. Documentation updated to reflect current PR.